src/YoutubeAtom.stories.tsx

```diff
@@ -376,3 +376,59 @@ export const MultipleVideos = (): JSX.Element => {
         </div>
     );
 };
+
+export const WithZeroDuration = (): JSX.Element => {
+    return (
+        <div style={containerStyle}>
+            <YoutubeAtom
+                elementId="xyz"
+                videoId="N9Cgy-ke5-s"
+                alt=""
+                role="inline"
+                eventEmitters={[
+                    (e) => console.log(`analytics event ${e} called`),
+                ]}
+                duration={0}
+                pillar={ArticlePillar.Opinion}
+                overrideImage={[
+                    {
+                        srcSet: [
+                            {
+                                src: 'https://i.guim.co.uk/img/media/4b3808707ec341629932a9d443ff5a812cf4df14/0_309_1800_1081/master/1800.jpg?width=1200&height=630&quality=85&auto=format&fit=crop&overlay-align=bottom%2Cleft&overlay-width=100p&overlay-base64=L2ltZy9zdGF0aWMvb3ZlcmxheXMvdGctZGVmYXVsdC5wbmc&enable=upscale&s=aff4b8255693eb449f13070df88e9cac',
+                                width: 1000,
+                            },
+                        ],
+                    },
+                ]}
```

Comment:
```suggestion
```

I guess we can remove this to simplify things?

Reply:
I think it changes the look of it, but let me check again